test/fixtures/microposts.yml

```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  content: MyText
+  user_id: 1
```

Comment:
What you did here is perfectly fine. But a "sweeter" way to do it is
```
user: one
```
This references the yaml key in the users fixtures file directly